### PR TITLE
Examples all take args

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/examples/java/io/nats/examples/NatsJsPushSub.java [hunk 1]
```diff
@@ -23,43 +23,45 @@
...
-    static final String DELIVER = null; // "push-sub-deliver";
-    static final int MSG_COUNT = 0;
-
     public static void main(String[] args) {
-        String server = ExampleArgs.getServer(args);
-        int count = MSG_COUNT < 1 ? Integer.MAX_VALUE : MSG_COUNT;
+        ExampleArgs exArgs = ExampleArgs.builder()
+                .defaultStream("example-stream")
+                .defaultSubject("example-subject")
+//                .defaultDurable("push-sub-durable")
```
Comment (ColinSullivan1) [inline src/examples/java/io/nats/examples/NatsJsPushSub.java:43]: Did you intend to leave in?

File: src/examples/java/io/nats/examples/NatsJsPushSubWithHandler.java [hunk 2]
```diff
@@ -25,47 +25,48 @@
...
-    static final String SUBJECT = "example-subject";
-    static final String DURABLE = "push-sub-handler-durable";
-    static final int MSG_COUNT = 3;
-
     public static void main(String[] args) {
-        String server = ExampleArgs.getServer(args);
+        ExampleArgs exArgs = ExampleArgs.builder()
+                .defaultStream("example-stream")
+                .defaultSubject("example-subject")
+//                .defaultDurable("push-sub-handler-durable")
```
Comment (ColinSullivan1) [inline src/examples/java/io/nats/examples/NatsJsPushSubWithHandler.java:44]: Same as above...

**Review by ColinSullivan1 — APPROVED** [review]
LGTM, Thanks.  Good to merge - up to you if you want to leave the commented options in or not.

File: src/examples/java/io/nats/examples/NatsJsPushSub.java (hunk 1 above)
Comment (scottf) [reply src/examples/java/io/nats/examples/NatsJsPushSub.java:43]: yes, in the instructions at top I mention that there is different behavior when durable

**Pull request merged**